article/sections/_3_graph_overview.tex

```diff
@@ -0,0 +1,160 @@
+\citeauthor{ben_2002_integration} draw a two-part process.
+First, they assume that explanatory variables cause unobserved utilities.
+Then, they assume that the unobserved utilities cause the choice.
+
+Note that although RUM diagrams adequately show the choice process, we still call them stylized.
+Specifically, these diagrams lack detail about the relationships between explanatory variables.
+When speaking collectively, we cannot tell if one explanatory variable causes another.
+Unfortunately, as shown in Section \ref{sec:graph-importance}, such knowledge is crucial.
+Without more detailed causal knowledge, our inferences may be inconsistent and arbitrarily bad.
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{iclv-causal-graph}
+   \caption{Archetypical ICLV causal diagram}
+   \label{fig:example-graph-iclv}
+\end{figure}
+
+
+Besides RUM models, causal graphs appear in the literature on Integrated Choice and Latent Variable (ICLV) models.
+An example of an ICLV model is in Figure \ref{fig:example-graph-iclv}, based on Figure 5 of \citet{ben_2002_integration}.
+As with RUM models, ICLV causal graphs represent assumptions about the choice process.
+Here concern typically centers around unobserved mediators.
+These unobserved variables cause the outcome and are caused by one's observed covariates.
+Omitting such mediators leads to models that misrepresent the assumed choice process.
+As a result, researchers care deeply about ICLV models that avoid these behavioral misrepresentations.
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{daysim-model-graph}
+   \caption{Archetypical causal diagram for activity-based models}
+   \label{fig:example-graph-abm}
+\end{figure}
+
+Finally, consider activity based travel demand models (ABM).
+These models often come with a causal diagram that depicts the interrelations between the outcomes in the model.
+E.g., household location choice, destination choice, travel mode choice, departure time choice, route choice etc.
+For example, see Figure \ref{fig:example-graph-abm}, from \citet[Fig. 2]{bradley_2010_sacsim}.
+The purpose of these graphs is to explain the structure of the entire system of outcome models.
+
+In particular, ABM diagrams highlight two sets of researcher assumptions.
+They detail the researcher's beliefs about which choices, i.e. outcomes, precede others.
+For instance, work location choice preceding auto-ownership choice.
+ABM diagrams also detail which downstream choices partially cause which preceding ones.
+For example, considerations about one's travel mode choice influences one's daily activity choices.
+This happens even though the activity choices precede the travel mode choices.
+
+Two main features distinguish RUM, ICLV, and ABM graphs from the causal graphs of \citet{pearl_1995_causal}.
+First, causal graphs in choice modelling traditionally ignore relations between the explanatory variables.
```

Comment:
Wondering if we should say "choice modeling graphs" instead of "causal graphs".

```suggestion
First, causal graphs in choice modelling traditionally do not depict relationships between their explanatory variables.
```